lib/WP_Auth0_LoginManager.php

```diff
@@ -144,12 +144,19 @@ public function login_auto() {
   }
 
   public function init_auth0() {
-    global $wp_query;
 
-    if ( $this->query_vars( 'auth0' ) === null ) {
+    // Not an Auth0 login process or settings are not configured to allow logins
+    if ( ! $this->query_vars( 'auth0' ) || ! WP_Auth0::ready() ) {
```

Comment:
`auth0` was the query param you use to track context, right?

Reply:
Yes, can be `1` for a "regular" login or `implicit`

Reply:
Hmm.. That's confusing, but as long as it's "internal" I don't care. 👍

Reply:
I agree but it's legacy and dangerous to unwire at this point. I'd like to just deprecate implicit altogether ...